### PR TITLE
Remove max-width and add hide text overflow

### LINKED ISSUE #27358: Issue Dependencies Overflow
### Description

Text overflows for Issue dependencies

### Screenshots

image

### Gitea Version

1.20 and 1.21

### Can you reproduce the bug on the Gitea demo site?

Yes

### Operating System

_No response_

### Browser Version

Chrome

### PR DESCRIPTION
**Pull request by kdumontnu**
Closes https://github.com/go-gitea/gitea/issues/27358

I'm happy for someone else to take this over if there is a more desirable solution, but this seemed like the simplest fix to me. 

Before:
image

After:
image

### CODE REVIEW

**Review by delvh — APPROVED** [review]

File: templates/repo/issue/view_content/sidebar.tmpl [hunk 1]
```diff
@@ -468,7 +468,7 @@
 									{{.Repository.OwnerName}}/{{.Repository.Name}}
 								</div>
 							</div>
-							<div class="item-right gt-df gt-ac">
+							<div class="item-right gt-df gt-ac  gt-m-2">
```
Comment (puni9869) [inline templates/repo/issue/view_content/sidebar.tmpl]:
```suggestion
							<div class="item-right gt-df gt-ac gt-m-2">
```

File: templates/repo/issue/view_content/sidebar.tmpl [hunk 2]
```diff
@@ -491,7 +491,7 @@
 										{{.Repository.OwnerName}}/{{.Repository.Name}}
 									</div>
 								</div>
-								<div class="item-right gt-df gt-ac">
+								<div class="item-right gt-df gt-ac  gt-m-2">
```
Comment (puni9869) [inline templates/repo/issue/view_content/sidebar.tmpl]:
```suggestion
								<div class="item-right gt-df gt-ac gt-m-2">
```

**Comment by puni9869** [comment]
could you apply the elipsis `...` for description.
image

I have one more suggestion: 
Add tooltip, this will retain the full name on hover.

**Comment by silverwind** [comment]
Yes, `...` ellipsis should be visible. The fact that it's not indicated there is something wrong with the styling.

**Comment by puni9869** [comment]
> Yes, `...` ellipsis should be visible. The fact that it's not indicated there is something wrong with the styling.


style was not applied

**Comment by kdumontnu** [comment]
@puni9869 @silverwind should be all set now.

Added tooltips. They are a bit off in terms of styling, which bugs me, but is out of the scope of this PR. 
ellipsis

**Review by techknowlogick — APPROVED** [review]
🙏 thanks!

**Pull request merged**